requirements.txt

```diff
@@ -2,3 +2,5 @@ gkeepapi
 keyring
 notion-client>=2.0.0
 requests<2.30.0
+notion
```

Comment:
I think this package is not used, as `notion-client` is still used.

Reply:
That's correct I believe it was because I was testing another script